test/src/com/xilinx/rapidwright/design/TestModuleInst.java

```diff
@@ -221,6 +223,61 @@ public void testConnectMovesPins() {
         Assertions.assertTrue(net2.getPins().isEmpty());
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = { "GND", "VCC" })
+    public void testConnectGNDVCC(String staticTypeName) {
```

Comment:
Does this test check both empty port insts and those that another net is already driving?